indico/modules/events/timetable/client/js/components/DetailsSegment.jsx

```diff
@@ -27,7 +27,7 @@ function ActionIcon({icon, onClick, title, color}) {
     <Icon
       name={icon}
       onClick={onClick}
-      style={{color: color?.text || 'rgba(0, 0, 0, 0.6)'}}
+      style={{color: color?.text || 'rgba(0, 0, 0, 0.6)', fontSize: 15}}
```

Comment:
is it possible to define this in a class? the only reason im even adding a style here is because we store colors as a hex. preferably, if we already have a stylesheet for the component, all of the styles should go there